jerry-core/ecma/builtin-objects/ecma-builtin-promise.c

```diff
@@ -578,9 +579,7 @@ ecma_builtin_promise_race_or_all (ecma_value_t this_arg, /**< 'this' argument */
   {
     ecma_raise_type_error (ECMA_ERR_MSG ("Second argument is not an array."));
```

Comment:
Anyway, it can be also moved. I've checked the [standard](https://www.ecma-international.org/ecma-262/6.0/#sec-ifabruptrejectpromise) and the raised error is the same in both cases (`Promise.rase/all`). OK, I move it as well.